docs/customization_guide/tritonfrontend.md

```diff
@@ -57,14 +57,18 @@ Note: `model_path` may need to be edited depending on your setup.
 
 2. Now, to start up the respective services with `tritonfrontend`
 ```python
-from tritonfrontend import KServeHttp, KServeGrpc
+from tritonfrontend import KServeHttp, KServeGrpc, Metrics
```

Comment:
I don't love that the `Metrics` object is a web server, so it makes me wonder if we should rename these down the line, ex: `KServeHttpService`, `MetricsService`, etc

But I don't have a strong opinion on an alternative right now so I think it's fine, just mentioning for later. We will probably be restructing some packaging and naming in the near-mid future.